api/capi/include/nntrainer.h

```diff
@@ -128,27 +140,27 @@ int ml_nnmodel_train_with_file(ml_nnmodel_h model, ...);
  * @retval #ML_ERROR_NONE Successful.
  * @retval #ML_ERROR_INVALID_PARAMETER Invalid Parameter.
  */
-int ml_nnmodel_train_with_generator(ml_nnmodel_h model,
+int ml_nnmodel_train_with_generator(ml_train_model_h model,
```

Comment:
Missing the prefix.

Reply:
Yes, this function is updated in #302 to control the size of this PR.